modules/indexer_monitors/versions.tf

```diff
@@ -0,0 +1,18 @@
+terraform {
+  cloud {
+    organization = "dydxprotocol"
+
+    workspaces {
+      tags = ["indexers"]
```

Comment:
nit: For this and `indexer_dashboards` can you use a different tag for the workspace? Using `indexers` means you can select any of `indexer-dev` / `indexer-staging` etc as a workspace.

Reply:
As a tangent, can you update this in the `indexer_dashboards` top-level directory? It shouldn't be taggedas  `indexers` since that top-level directory could potentially plan on `indexer-dev` / `indexer-staging` etc.

Reply:
done